docs/develop/01_sdk/02_cookbook/05_messaging/01_messaging.md

```diff
@@ -0,0 +1,53 @@
+---
+id: messaging_book
+title: Generate a Message
+---
+
+import SnippetBlock from '@site/src/components/SnippetBlock';
+
+import GenerateMessage from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/messaging/01_generate_message.ts';
+import EncryptMessage from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/messaging/02_encrypt_message.ts';
+import DecryptMessage from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/messaging/03_decrypt_message.ts';
+
+KILT uses a custom [Messaging protocol](../../../../concepts/06_messaging.md) that enables users of the KILT protocol to communicate via [unicast network](https://en.wikipedia.org/wiki/Unicast).
```

Comment:
I would only say that. Otherwise it sounds like KILT is a messaging app.

```suggestion
KILT defines a [unicast](https://en.wikipedia.org/wiki/Unicast) [messaging protocol](../../../../concepts/06_messaging.md).
```